library/billing/src/BillingUtilities.php

```diff
@@ -1279,17 +1279,17 @@ class BillingUtilities
 
     public static function getBillingById($id, $cols = "*")
     {
-        return sqlQuery("select $cols from billing where id=? and activity=1 order by date DESC limit 0,1", array($id));
+        return sqlQuery("select " . escape_sql_column_name($cols, array('billing')) . " from billing where id=? and activity=1 order by date DESC limit 0,1", array($id));
     }
 
     public static function getBillingByPid($pid, $cols = "*")
```

Comment:
This function can be completely removed since not used anywhere in OpenEMR's codebase :)